src/components/UI/RefreshableLabel.js

```diff
@@ -59,13 +59,12 @@ export function RefreshableLabel({ children, value, style }) {
 
   return (
     <Wrapper className={hasDataChanged && 'changed'} style={style}>
-      {children}
+      {this.props.children}
```

Comment:
This is a functional component, so `this` is not the same as in a class component, it doesn't have the `props` property. If you don't want to destructure it, you can use the rest operator in the compoonent props

Example:
```js
export function RefreshableLabel({ children, value, style, ...props })
```

and then use `props.children` as you would normally.

Reply:
Thanks for the explanation, I guess it was a lapse :-P